wallet/src/de/schildbach/wallet/service/platform/TopUpRepository.kt

```
@@ -311,4 +366,283 @@ class TopUpRepositoryImpl @Inject constructor(
+                        "",
+                        blockTimestamp,
+                        0
+                    )
+
+                    // profile information here
+                    try {
+                        if (platformRepo.updateDashPayProfile(identityId)) {
+                            val profile = dashPayProfileDao.loadByUserId(identityId)
+                            invite = invite.copy(acceptedAt = profile?.createdAt ?: -1) // it was accepted in the past, use profile creation as the default
+                        }
+                    } catch (e: NullPointerException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)